scripts/create_addon

```diff
@@ -113,12 +113,12 @@ pack_addon() {
       if [ "$ADDON_OVERWRITE" = "yes" ]; then
         rm $ADDON_INSTALL_DIR/$PKG_ADDON_ID-$ADDONVER.zip
       else
-        printf  "%${BUILD_INDENT}c $(print_color CLR_WARNING "*** WARNING: $PKG_ADDON_ID-$ADDONVER.zip already exists. not overwriting it ***")\n" ' '>&$SILENT_OUT
+        build_msg "CLR_WARNING" "" "*** WARNING: ${PKG_ADDON_ID}-${ADDONVER}.zip already exists. Not overwrittng it. ***"
```

Comment:
```suggestion
        build_msg "CLR_WARNING" "*** WARNING: ${PKG_ADDON_ID}-${ADDONVER}.zip already exists. Not overwrittng it. ***" ""
```
as we want the message text to be output using the "warning" colour (ie. red).


We may want to check arg3 to `build_msg()` and use a slightly different printf when it is blank. For example:
```
-  printf "%${BUILD_INDENT}c$(print_color "${1}" "${2}")      ${3}\n" ' '>&${SILENT_OUT} 
+  if [ -n "${3}" ]; then
+    printf "%${BUILD_INDENT}c$(print_color "${1}" "${2}")      ${3}\n" ' '>&${SILENT_OUT}
+  else
+    printf "%${BUILD_INDENT}c$(print_color "${1}" "${2}")\n" ' '>&${SILENT_OUT}
+  fi
```